src/gt4py/next/program_processors/runners/dace_fieldview/gtir_dataflow.py

```diff
@@ -31,18 +31,42 @@
 
 @dataclasses.dataclass(frozen=True)
 class DataExpr:
```

Comment:
What is the difference between this class and `Field`, they have the same members.
I am not against that per se, if they are used in different contexts and the name provides more hints about what is going on.
But, in that case there _must_ be a detailed description what differentiates this class from the other.

Reply:
`DataExpr` is used in the context in a dataflow, a.k.a. a stencil expression. Therefore, it either represent scalar values (elements in the fields) or lists of local values. The other type (now renamed to `FieldopData`) represents the result of field operators, basically the data storage outside global maps.